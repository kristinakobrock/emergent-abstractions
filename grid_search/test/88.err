  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  4%|###7                                                                                         | 2/50 [00:00<00:02, 17.44it/s]  8%|#######4                                                                                     | 4/50 [00:00<00:02, 18.75it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:02, 18.88it/s] 18%|################7                                                                            | 9/50 [00:00<00:02, 19.93it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 22.85it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 22.49it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 24.35it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 27.18it/s] 50%|##############################################                                              | 25/50 [00:01<00:00, 27.49it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.02it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 23.93it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 23.10it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 22.62it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 24.58it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.57it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.28it/s]100%|############################################################################################| 50/50 [00:02<00:00, 26.20it/s]100%|############################################################################################| 50/50 [00:02<00:00, 24.18it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 31%|############################6                                                                | 4/13 [00:00<00:00, 33.04it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 24.74it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 23.48it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.93it/s]
