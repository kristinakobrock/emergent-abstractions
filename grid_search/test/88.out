# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6666092276573181, "acc": 0.623019278049469, "length": 3.534140110015869, "mode": "train", "epoch": 1}
{"loss": 0.668973982334137, "acc": 0.6294495463371277, "length": 1.0078125, "mode": "test", "epoch": 1}
{"loss": 0.661810576915741, "acc": 0.6244471669197083, "length": 3.50703501701355, "mode": "train", "epoch": 2}
{"loss": 0.6645828485488892, "acc": 0.6333984732627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6611148118972778, "acc": 0.6256840825080872, "length": 3.5140926837921143, "mode": "train", "epoch": 3}
{"loss": 0.6731655597686768, "acc": 0.6181488037109375, "length": 1.015625, "mode": "test", "epoch": 3}
{"loss": 0.6607755422592163, "acc": 0.6256366968154907, "length": 3.4973835945129395, "mode": "train", "epoch": 4}
{"loss": 0.6589397192001343, "acc": 0.6326843500137329, "length": 1.4765625, "mode": "test", "epoch": 4}
{"loss": 0.6589669585227966, "acc": 0.6256910562515259, "length": 3.4847965240478516, "mode": "train", "epoch": 5}
{"loss": 0.6534534096717834, "acc": 0.6305633783340454, "length": 3.1015625, "mode": "test", "epoch": 5}
{"loss": 0.6586162447929382, "acc": 0.6270978450775146, "length": 3.5105221271514893, "mode": "train", "epoch": 6}
{"loss": 0.6563246250152588, "acc": 0.6283599734306335, "length": 3.9375, "mode": "test", "epoch": 6}
{"loss": 0.6565390825271606, "acc": 0.6263973712921143, "length": 3.5168564319610596, "mode": "train", "epoch": 7}
{"loss": 0.6481113433837891, "acc": 0.631640613079071, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.6521811485290527, "acc": 0.6267663240432739, "length": 3.6020920276641846, "mode": "train", "epoch": 8}
{"loss": 0.6457968950271606, "acc": 0.640429675579071, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6427488923072815, "acc": 0.6380845904350281, "length": 3.64266037940979, "mode": "train", "epoch": 9}
{"loss": 0.6391328573226929, "acc": 0.6322265267372131, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6253262758255005, "acc": 0.6637457013130188, "length": 3.751081705093384, "mode": "train", "epoch": 10}
{"loss": 0.6347540616989136, "acc": 0.6384764313697815, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.6064361929893494, "acc": 0.6750925779342651, "length": 3.8421266078948975, "mode": "train", "epoch": 11}
{"loss": 0.6452178955078125, "acc": 0.6556640267372131, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.5996468663215637, "acc": 0.6806896328926086, "length": 3.9239070415496826, "mode": "train", "epoch": 12}
{"loss": 0.6549580097198486, "acc": 0.6578125357627869, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5935738682746887, "acc": 0.6875899434089661, "length": 3.948881149291992, "mode": "train", "epoch": 13}
{"loss": 0.6505029797554016, "acc": 0.65625, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5890024304389954, "acc": 0.6901424527168274, "length": 3.9556326866149902, "mode": "train", "epoch": 14}
{"loss": 0.6559637188911438, "acc": 0.6527343392372131, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5859513878822327, "acc": 0.6903287172317505, "length": 3.967651605606079, "mode": "train", "epoch": 15}
{"loss": 0.662285566329956, "acc": 0.6603515148162842, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5808215141296387, "acc": 0.6970916986465454, "length": 3.9726619720458984, "mode": "train", "epoch": 16}
{"loss": 0.6579886674880981, "acc": 0.654101550579071, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5806409120559692, "acc": 0.695953369140625, "length": 3.9790756702423096, "mode": "train", "epoch": 17}
{"loss": 0.657177209854126, "acc": 0.6572265625, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5789409279823303, "acc": 0.6984601020812988, "length": 3.985700845718384, "mode": "train", "epoch": 18}
{"loss": 0.6423627138137817, "acc": 0.6591796875, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5756081342697144, "acc": 0.6981856822967529, "length": 3.987658739089966, "mode": "train", "epoch": 19}
{"loss": 0.6314623355865479, "acc": 0.6628904938697815, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5730811953544617, "acc": 0.69877028465271, "length": 3.9874610900878906, "mode": "train", "epoch": 20}
{"loss": 0.6391022205352783, "acc": 0.6554687023162842, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.5716584324836731, "acc": 0.6997764110565186, "length": 3.989537000656128, "mode": "train", "epoch": 21}
{"loss": 0.6177279949188232, "acc": 0.663281261920929, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.5694442987442017, "acc": 0.7034285664558411, "length": 3.991123676300049, "mode": "train", "epoch": 22}
{"loss": 0.6203956604003906, "acc": 0.6677734851837158, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.5663226246833801, "acc": 0.7040571570396423, "length": 3.987409830093384, "mode": "train", "epoch": 23}
{"loss": 0.6129326820373535, "acc": 0.6785155534744263, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.5664121508598328, "acc": 0.7037360072135925, "length": 3.9928338527679443, "mode": "train", "epoch": 24}
{"loss": 0.6106876730918884, "acc": 0.6779298186302185, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.563477098941803, "acc": 0.7089275121688843, "length": 3.9901397228240967, "mode": "train", "epoch": 25}
{"loss": 0.6099091172218323, "acc": 0.6705078482627869, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.5634967088699341, "acc": 0.7044835090637207, "length": 3.9905099868774414, "mode": "train", "epoch": 26}
{"loss": 0.6079001426696777, "acc": 0.6707029938697815, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.5591307282447815, "acc": 0.7082624435424805, "length": 3.985257863998413, "mode": "train", "epoch": 27}
{"loss": 0.5997079610824585, "acc": 0.6812500357627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.5552867650985718, "acc": 0.7107746005058289, "length": 3.985417604446411, "mode": "train", "epoch": 28}
{"loss": 0.5930320024490356, "acc": 0.6912108659744263, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.5508575439453125, "acc": 0.7156994342803955, "length": 3.9879019260406494, "mode": "train", "epoch": 29}
{"loss": 0.5898692011833191, "acc": 0.6925781965255737, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.544191300868988, "acc": 0.722667396068573, "length": 3.9865224361419678, "mode": "train", "epoch": 30}
{"loss": 0.5845109224319458, "acc": 0.6927734017372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.5377864241600037, "acc": 0.725247323513031, "length": 3.9889698028564453, "mode": "train", "epoch": 31}
{"loss": 0.5828474760055542, "acc": 0.6986327767372131, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.53364497423172, "acc": 0.7295036315917969, "length": 3.9826176166534424, "mode": "train", "epoch": 32}
{"loss": 0.5858058333396912, "acc": 0.6941404938697815, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.5265647768974304, "acc": 0.7326211333274841, "length": 3.9869654178619385, "mode": "train", "epoch": 33}
{"loss": 0.5780664682388306, "acc": 0.6908202767372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.5203362107276917, "acc": 0.7374835014343262, "length": 3.993504047393799, "mode": "train", "epoch": 34}
{"loss": 0.5821444988250732, "acc": 0.6941404938697815, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.5141724348068237, "acc": 0.7435017824172974, "length": 3.9936351776123047, "mode": "train", "epoch": 35}
{"loss": 0.5685768127441406, "acc": 0.7089844346046448, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.510483980178833, "acc": 0.7451159954071045, "length": 3.9960784912109375, "mode": "train", "epoch": 36}
{"loss": 0.5609717965126038, "acc": 0.7068359851837158, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.509192943572998, "acc": 0.744572103023529, "length": 3.9949278831481934, "mode": "train", "epoch": 37}
{"loss": 0.5558339357376099, "acc": 0.7150391340255737, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.5042939782142639, "acc": 0.750237762928009, "length": 3.99220871925354, "mode": "train", "epoch": 38}
{"loss": 0.5536277890205383, "acc": 0.712890625, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.49854525923728943, "acc": 0.7529746294021606, "length": 3.99579119682312, "mode": "train", "epoch": 39}
{"loss": 0.5555840730667114, "acc": 0.7119140625, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.4928085505962372, "acc": 0.7544864416122437, "length": 3.994913339614868, "mode": "train", "epoch": 40}
{"loss": 0.5450791716575623, "acc": 0.7195312976837158, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.48957914113998413, "acc": 0.7570478320121765, "length": 3.9957401752471924, "mode": "train", "epoch": 41}
{"loss": 0.559660792350769, "acc": 0.7253906726837158, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.48597830533981323, "acc": 0.7633761763572693, "length": 3.9928207397460938, "mode": "train", "epoch": 42}
{"loss": 0.5439838171005249, "acc": 0.7179687023162842, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.482745498418808, "acc": 0.7643647789955139, "length": 3.996603488922119, "mode": "train", "epoch": 43}
{"loss": 0.5451434254646301, "acc": 0.7238280773162842, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.47342732548713684, "acc": 0.772843599319458, "length": 3.996776819229126, "mode": "train", "epoch": 44}
{"loss": 0.5328055620193481, "acc": 0.7447265982627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.4678354859352112, "acc": 0.7744181156158447, "length": 3.996882438659668, "mode": "train", "epoch": 45}
{"loss": 0.542291522026062, "acc": 0.7355468273162842, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.46403399109840393, "acc": 0.7769710421562195, "length": 3.992497444152832, "mode": "train", "epoch": 46}
{"loss": 0.5303552150726318, "acc": 0.7371092438697815, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.46132728457450867, "acc": 0.7811836004257202, "length": 3.9972589015960693, "mode": "train", "epoch": 47}
{"loss": 0.5233830809593201, "acc": 0.7478514909744263, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.4522116482257843, "acc": 0.7882207036018372, "length": 3.9946486949920654, "mode": "train", "epoch": 48}
{"loss": 0.5177929401397705, "acc": 0.7574219107627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.45049765706062317, "acc": 0.7887644171714783, "length": 3.994699478149414, "mode": "train", "epoch": 49}
{"loss": 0.5169431567192078, "acc": 0.7574219107627869, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.44478192925453186, "acc": 0.7960729598999023, "length": 3.9966037273406982, "mode": "train", "epoch": 50}
{"loss": 0.5109292268753052, "acc": 0.7587890625, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.4393106997013092, "acc": 0.7969749569892883, "length": 3.9990811347961426, "mode": "train", "epoch": 51}
{"loss": 0.5066089630126953, "acc": 0.7628907561302185, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.4345560669898987, "acc": 0.80182284116745, "length": 3.9965875148773193, "mode": "train", "epoch": 52}
{"loss": 0.5100181102752686, "acc": 0.7626953125, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.43145403265953064, "acc": 0.8031566739082336, "length": 3.993820905685425, "mode": "train", "epoch": 53}
{"loss": 0.5062214136123657, "acc": 0.7613281607627869, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.42724576592445374, "acc": 0.8086766004562378, "length": 3.9971866607666016, "mode": "train", "epoch": 54}
{"loss": 0.5074323415756226, "acc": 0.7654296159744263, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.4251721501350403, "acc": 0.8082918524742126, "length": 3.9959492683410645, "mode": "train", "epoch": 55}
{"loss": 0.5046324729919434, "acc": 0.7673828601837158, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.42195942997932434, "acc": 0.810910701751709, "length": 3.995659828186035, "mode": "train", "epoch": 56}
{"loss": 0.5057197213172913, "acc": 0.7669921517372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.41628095507621765, "acc": 0.813069760799408, "length": 3.997356653213501, "mode": "train", "epoch": 57}
{"loss": 0.5046550631523132, "acc": 0.7630859017372131, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.4118216335773468, "acc": 0.8169198632240295, "length": 3.9990108013153076, "mode": "train", "epoch": 58}
{"loss": 0.49685585498809814, "acc": 0.7742187976837158, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.4125272333621979, "acc": 0.8180028200149536, "length": 3.9970383644104004, "mode": "train", "epoch": 59}
{"loss": 0.5083054304122925, "acc": 0.7722656726837158, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.4112268388271332, "acc": 0.8179484605789185, "length": 3.9963622093200684, "mode": "train", "epoch": 60}
{"loss": 0.5010733604431152, "acc": 0.7736327648162842, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7010343670845032
